custom/icds_reports/models/aggregate.py

```diff
@@ -848,6 +848,10 @@ class DailyAttendance(models.Model, AggregateMixin):
     image_name = models.TextField(null=True)
     pse_conducted = models.SmallIntegerField(null=True)
     state_id = models.TextField(null=True)
+    open_bfast_count = models.IntegerField(null=True)
+    open_hotcooked_count = models.IntegerField(null=True)
+    days_thr_provided_count = models.IntegerField(null=True)
+    open_pse_count = models.IntegerField(null=True)
```

Comment:
Is this different than the pse_conducted column that already exists?

Reply:
Yep, a bit. pse_conducted is when awc conducted some activity and this when at least one kid was present